packages/manager/apps/web/client/app/email-domain/upgrade/email-domain.service.js

```diff
@@ -0,0 +1,55 @@
+import { map } from '@uirouter/core';
```

Comment:
it seems weird to me to import map from uirouter/core, why not lodash ?

Reply:
oh, it was not intentional. But it worked. Anyway, I have changed it. THANKS